source/clear-linux/tutorials/greengrass.rst

```diff
@@ -100,7 +101,7 @@ for the OpenVINO software stack:
 
    Learn more about how to :ref:`swupd-guide`.
 
-The ``computer-vision-basic`` bundle will install the OpenVINO software,
+The ``computer-vision-basic`` bundle installs the OpenVINO™ toolkit,
 along with the edge device models needed.
```

Comment:
"... installs the OpenVINO™ toolkit and the sample models optimized for Intel® edge platforms"